Holovibes/includes/core/compute_settings.hh

```diff
@@ -4,32 +4,39 @@
  */
 #pragma once
 
-#include "API.hh"
+#include <nlohmann/json_fwd.hpp>
+using json = ::nlohmann::json;
```

Comment:
Maybe a blank line between the two ? As includes and usings are not the same things